echopype/tests/echodata/test_zarr_combine.py

```diff
@@ -300,3 +327,101 @@ def test_append_ds_list_to_zarr(append_ds_list_params):
     temp_zarr_dir.cleanup()
 
 
+class TestZarrCombine:
+    # initiate ZarrCombine object
+    zarr_combine = ZarrCombine()
+    sonar_model = "EK60"
+
+    def test_constructor(self):
+        # all possible time dimensions
+        assert self.zarr_combine.possible_time_dims == {
+            "time1",
+            "time2",
+            "time3",
+            "ping_time",
+        }
+        # all possible dimensions that we will append to (mainly time dims)
+        assert self.zarr_combine.append_dims == {"filenames"}.union(self.zarr_combine.possible_time_dims)
+        # encodings associated with lazy loaded variables
+        assert self.zarr_combine.lazy_encodings == ["chunks", "preferred_chunks"]
+        # defaultdict that holds every group's attributes
+        assert self.zarr_combine.group_attrs == defaultdict(list)
+
+        # The sonar_model for the new combined EchoData object
+        assert self.zarr_combine.sonar_model is None
+
+        # The maximum chunk length allowed for every append dimension
+        assert self.zarr_combine.max_append_chunk_size == 1000
+
+        # initialize variables created within class methods
+        # descriptions of these variables can be found in _get_ds_info
+        assert self.zarr_combine.dims_df is None
+        assert self.zarr_combine.dims_sum is None
+        assert self.zarr_combine.dims_csum is None
+        assert self.zarr_combine.dims_max is None
+
+        # Ensure that combine exists
+        assert hasattr(self.zarr_combine, 'combine')
+
+    @pytest.mark.parametrize("consolidated", [True, False])
+    def test_combine_consolidated(self, ek60_test_data, consolidated):
+        zarr_combine = ZarrCombine()
+        eds = [
+            echopype.open_raw(raw_file=file, sonar_model=self.sonar_model)
+            for file in ek60_test_data
+        ]
+        # create temporary directory for zarr store
+        temp_zarr_dir = tempfile.TemporaryDirectory()
+        zarr_file_name = temp_zarr_dir.name + f"/combined_echodatas_{str(consolidated)}.zarr"
+
+        zarr_path = check_zarr_path(zarr_file_name)
+
+        _, echodata_filenames = check_echodatas_input(eds)
+
+        # create dask client
+        client = Client()
+
+        # combined = echopype.combine_echodata(eds, zarr_file_name, client=client)
+        # combine all elements in echodatas by writing to a zarr store
+        _ = zarr_combine.combine(
+            zarr_path,
+            eds,
+            sonar_model=self.sonar_model,
+            echodata_filenames=echodata_filenames,
+            consolidated=consolidated,
+        )
+
+        check = True if consolidated else False
+        zmeta_path = Path(zarr_path) / ".zmetadata"
+
+        assert zmeta_path.exists() is check
+
+        if check is True:
```

Comment:
All of my comments from `test_echodata.py` should apply here. From what I can see this is just a copy paste of items in `test_to_zarr_consolidated` could we make this into a function and put it into say `utils.py`?